---
date: 2019-06-04
title: Lam 2018
categories:
  - Klebsiella pneumoniae
description: AMR genes detection and genomic epidemiology of hypervirulent Klebsiella pneumoniae
type: Document
related_post:
comments: true

---


* Genomics and AMR genes of emerging Klebsiella pneumoniae strains using short and long reads



## Summary table

|Key|Value|
|----|----|
|category|assembly-based|
|platform|illumina/ MinION|
|open source|true|
|reproducible|true|


## Database

ARG-Annot database in Kleborate


## Protocol


* Step 1

Genomic DNA was sequenced using the MiniSeq or MiSeq (Illumina) and MinION (Oxford Nanopore) 

* Step 2

SPAdes v3.8.1 was used for de novo assembly of short reads data  
Unicycler v0.4.4 was used for hybrid assembly of short and long reads data

* Step 3 

Antimicrobial resistance genes were searched through the database using thresholds of ≥ 90.0% for both nucleotide identity and sequence coverage.

## Citation

The paper(s) documenting this protocol:  
*Lam M et al. [Population genomics of hypervirulent Klebsiella pneumoniae clonal-group 23 reveals early emergence and rapid global dissemination. Nature Communications. 2018](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6045662/)  
*Wyres KL et al. [Emergence and rapid global dissemination of CTX-M-15-associated Klebsiella pneumoniae strain ST307. Journal of Antimicrobial Chemotherapy. 2018](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6376852/)

